codalab/worker/bundle_manager.py

```diff
@@ -111,6 +112,10 @@ def _stage_bundles(self):
         bundles_to_fail = []
         bundles_to_stage = []
         for bundle in bundles:
+            # Ignore non-derived (i.e. executable) bundles
+            if not isinstance(bundle, DerivedBundle):
```

Comment:
non-derived is not-executable (comment is ambiguous)

Do we need this statement?